alfalfa_web/components/App/App.js

```diff
@@ -61,8 +61,9 @@ class App extends React.Component {
             </AppBar>
             <Routes>
               <Route path="/" element={<Upload />} />
-              <Route path="models" element={<Sites />} />
-              <Route path="sims" element={<Sims />} />
+              <Route path="/models" element={<Sites />} />
+              <Route path="/sims" element={<Sims />} />
```

Comment:
Out of scope for this PR, but it would be nice if we could align the terminology/paths with the vocab we use in other parts of alfalfa.